src/components/header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const Header = () => {
+  return (
+    <header>
+        <nav className="navbar navbar-default  navbar-fixed-top">
+            <div className="container-fluid">
+              <div className="navbar-header">
+                  <a className="navbar-brand" href="/">
+                    <img src="https://www.okta.com/sites/all/themes/Okta/images/logo.svg" height="100%" width="100%"/>
```

Comment:
not sure if we should host our images on www.okta.com - why not just pull it down and serve it up locally? i.e. this demo wouldn't look correct if we were not on the internet. Same goes for other resources we're not serving up locally.